internal/cluster/transition.go

```diff
@@ -105,8 +105,16 @@ func (th *transitionHandler) PostPrepareForInstallation(sw stateswitch.StateSwit
 
 	sendNTPMetric(logutil.FromContext(params.ctx, th.log), params.metricApi, sCluster.cluster)
 
-	if err := params.manifestsGenerator.AddIpv6Manifest(params.ctx, logutil.FromContext(params.ctx, th.log), sCluster.cluster); err != nil {
-		return errors.Wrap(err, "PostPrepareForInstallation failed to add ipv6 manifest")
+	if len(sCluster.cluster.Hosts) == 1 {
+		ipv6OnlyHost, err := network.IsIpv6OnlyHost(sCluster.cluster.Hosts[0], th.log)
+		if err != nil {
+			return errors.Wrap(err, "Getting ipv6 infromation on host")
```

Comment:
The error message doesn't look like an error. Maybe "Failed to get IPv6 information on host" or something. Maybe even include the host ID